saleor/graphql/attribute/mutations.py

```diff
@@ -247,7 +247,34 @@ class AttributeUpdateInput(graphene.InputObjectType):
     )
 
 
+class AttributeProperties:
+    FILTERABLE_IN_STOREFRONT = "filterable_in_storefront"
+    FILTERABLE_IN_DASHBOARD = "filterable_in_dashboard"
+    AVAILABLE_IN_GRID = "available_in_grid"
+    STOREFRONT_SEARCH_POSITION = "storefront_search_position"
+
+
 class AttributeMixin:
+    INPUT_TYPE_TO_NOT_ALLOWED_ATTRIBUTE_PROPERTIES = {
```

Comment:
Maybe it would be easier to maintain a list of properties that are allowed per input type? If an input type is not present, all properties are allowed, if it is present here, the list says which properties are allowed. I would also think of a simpler name, maybe `ATTRIBUTE_PROPERTIES_CONFIGURATION`?

Reply:
I changed and moved it to `/attribute/__init__.py`